obc.py

```diff
@@ -23,7 +23,7 @@ def start(self):
 
         # start apps
         imageService.start()
-        telemData.start()
+        pixhawk.start()
 
 
```

Comment:
Nit-pick: You've got a couple extra lines of white space between these two functions. Normally, there should only be 1

Reply:
That's a relief. I can go through and fix that right away.